base/Dockerfile

```diff
@@ -47,15 +45,11 @@ RUN --mount=type=secret,id=repocred,target=/etc/apt/auth.conf.d/kopano.conf \
         moreutils \
         python3-minimal \
         && \
-    rm -rf /var/cache/apt /var/lib/apt/lists/*; \
-    # install apt key if supported kopano
-    if [ ${RELEASE_KEY_DOWNLOAD} -eq 1 ]; then \
-        if [ "$(awk 'BEGIN { FS="[ ]" } ; { print $6 }' < /etc/apt/auth.conf.d/kopano.conf)" != "REPLACE-ME" ]; then \
-            curl -u serial:"$(awk 'BEGIN { FS="[ ]" } ; { print $6 }' < /etc/apt/auth.conf.d/kopano.conf)" -s -S -o - "${KOPANO_CORE_REPOSITORY_URL}/Release.key" | apt-key add -; \
-        else \
-            curl -s -S -o - "${KOPANO_CORE_REPOSITORY_URL}/Release.key" | apt-key add -; \
-        fi \
-    fi
+    rm -rf /var/cache/apt /var/lib/apt/lists/*
+
+# hadolint ignore=SC2215
+RUN --mount=type=bind,source=./Release.key,target=/tmp/kopano-release.key \
```

Comment:
What I meant to say is to simply add the file to the container in general. No need to handle it as a secret.

Reply:
@fbartels So first a COPY and then RUN apt-key add ? Or is there a better way?